admin-ui/src/test/java/io.crate/rest.action.admin/AdminUIIntegrationTest.java

```diff
@@ -56,6 +56,27 @@ public void testLegacyRedirect() throws IOException, URISyntaxException {
     }
 
     @Test
+    public void testPluginURLRedirect() throws IOException, URISyntaxException {
+        //request to '/_plugin/crate-admin' is redirected to '/index.html'
+        Header[] headers = {
+            browserHeader()
+        };
+
+        List<URI> allRedirectLocations = getAllRedirectLocations("/_plugin/crate-admin", headers);
+
+        // all redirect locations should not be null
+        assertThat(allRedirectLocations, notNullValue());
```

Comment:
remove, unnecessary, contains assertion will fail if its null, it's just confusing ;)